backend/infrastructures/apps/extract/views/views.py

```diff
@@ -0,0 +1,103 @@
+import logging
+
+import inject
+from drf_spectacular import utils as swagger_utils
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from modules.common import const as common_consts
+from modules.common.domain import exceptions as domain_exceptions
+from modules.data.domain import value_objects as data_value_objects
+from modules.extract.domain import commands as domain_extract_commands
+from modules.extract.domain.ports import units_of_work as extract_units_of_work
+from modules.extract.services import commands as service_extract_commands
+from modules.load.domain import commands as domain_load_commands
+from modules.load.domain.ports import units_of_work as load_units_of_work
+from modules.load.services import commands as services_load_commands
+from modules.load.services import queries as load_queries
+from modules.transform.domain import commands as domain_transform_commands
+from modules.transform.services import commands as services_transform_commands
+
+logger = logging.getLogger(__name__)
+
+
+class ExtractViewSet(
+    ViewSet,
+):
+    @swagger_utils.extend_schema(
+        responses={
+            status.HTTP_201_CREATED: swagger_utils.OpenApiResponse(
```

Comment:
@taralalukasz 
Please make sure that swagger documentation matches real view possible responses.